### ISSUE TITLE
Add twitch player to npm

### ISSUE THREAD

**Issue by AStaroverov** [justintv/Twitch-API#539]
Add [this](http://player.twitch.tv/js/embed/v1.js) at npm

**Comment by mavrick** [comment]
+1

**Comment by tony99nyr** [comment]
I needed this too, so I just created one.  I believe this closes this issue.  The twitch video embed script is now available on npm. `npm i twitch-embed`

https://github.com/tony99nyr/twitch-embed

**Comment by AStaroverov** [comment]
Unfortunately, it's not a complete solution

чт, 28 июл. 2016 г. в 0:14, Tony Rossi <notifications@github.com>:

**Comment by tony99nyr** [comment]
You are correct.  It's not the best solution, but it is the best I can do without the source. I added an example of how I am using it in React.  It's enough to get the job done on my project.

**Comment by AStaroverov** [comment]
Thx for this!)

чт, 28 июл. 2016 г. в 15:47, Tony Rossi <notifications@github.com>:
